huaweicloud/services/dli/resource_huaweicloud_dli_queue_v1.go

```diff
@@ -0,0 +1,235 @@
+package dli
+
+import (
+	"fmt"
+	"regexp"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+	"github.com/huaweicloud/golangsdk/openstack/common/tags"
+	"github.com/huaweicloud/golangsdk/openstack/dli/v1/queues"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils/fmtp"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils/logp"
+)
+
+var regexp4Name = regexp.MustCompile(`^[a-z0-9_]{1,128}$`)
+
+const CU_16, CU_64, CU_256 = 16, 64, 256
+const RESOURCE_MODE_SHARED, RESOURCE_MODE_EXCLUSIVE = 0, 1
+
+func ResourceDliQueueV1() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceDliQueueCreate,
+		Read:   resourceDliQueueV1Read,
+		Delete: resourceDliQueueV1Delete,
+
+		Schema: map[string]*schema.Schema{
```

Comment:
We should keep `management_subnet cidr`, `subnet cidr` and `vpc cidr` compatible with previous versions.